apollo-router/src/plugins/telemetry/config_new/fixtures/router/http.server.request.duration/router.yaml

```diff
@@ -3,4 +3,5 @@ telemetry:
     instruments:
       router:
         http.server.active_requests: false
-        http.server.request.duration: true
\ No newline at end of file
+        http.server.request.duration: true
+        http.server.response.body.size: false
```

Comment:
Is there a reason these are going from `true` to `false`?

Reply:
The default requirement level (Required) enabled them by default, they didn't show up before as content-length was not specified in the test yaml but since we now do it based on the body it would show up if not disabled here. I figured it would be best to disable it for the tests not relating to response body size so it doesn't show up in the snapshots